composer.json

```diff
@@ -31,6 +31,7 @@
         "mongodb/mongodb": "^1.15"
     },
     "require-dev": {
+        "mongodb/builder": "^0.2",
```

Comment:
We could add this to `suggest` to call out that it's necessary to use the agg builder.